test/unit_tests/braket/default_simulator/openqasm/test_interpreter.py

```diff
@@ -982,8 +982,10 @@ def test_pow():
 
 def test_measurement_noop_does_not_raise_exceptions():
     qasm = """
-    qubit q;
-    measure q;
+    qubit[2] q;
+    bit[1] c;
+    measure q[0];
+    c = measure q[1];
```

Comment:
Is there a reason for this change? Are there any asserts that this test should have?

Reply:
To make sure no errors are thrown is enough. This test was modified to make sure no errors are thrown both for regular measurements and those that assign to variables.